### DISCUSSION TITLE
Add GIF format for previewing projects.

### DISCUSSION THREAD

**Discussion by Agnyy** [opentoonz/opentoonz#6353] [Ideas]
Somehow I started making a project for OT with animated workpieces. There are transitions and stuff. 
https://agnyy.itch.io/big-pack-for-opentoonzI 
https://www.youtube.com/watch?v=dFeX4Krk_SQ (Video)
peeped at this idea from my professional experience. When you need to do quickly and many projects, we often use blanks.
![image](https://user-images.githubusercontent.com/25137502/84405785-4a592880-ac11-11ea-974a-8dff80ef4b9b.png) 
 But in OT there is a big minus for using this approach is a long loading preview. I suggest adding a GIF format for previewing projects, then a gif instead of a project will be loaded in the window, this will significantly speed up the speed of the preview. So do for projects in AE. https://videohive.net/item/big-pack-of-elements/19888878

**Comment by RodneyBaker** [comment]
I commented in discord where your posted this idea.

There I mention that an obstacle to being able to view gif animations (and gif stills as well) is that currently the only way Opentoonz processes gif animation is through ffmpeg and ffmpeg doesn't process anything for the Opentoonz UI.

I'm not sure how to get around this obstacle short of creating a new method of processing GIF file in Opentoonz.

I love the idea of being able to see gif animation via the Browser.
How to get there is the main concern.

**Comment by RodneyBaker** [comment]
It seems that  `QTreeWidget::setItemWidget()` with the gif being processed through `QMovie` has worked for some people.
